src/Engine/Systems/Subsystems/PhysicsRenderBridge.hpp

```
@@ -0,0 +1,46 @@
+#include <mutex>
+#include <queue>
+
+#include <Core/Data/TripleBuffer.hpp>
+
+#include <Engine/Rendering/Data/Buffer.hpp>
+
+#include <Platform/External/GLM.hpp>
+
+
+/* Data handoff mechanism between PhysicsSystem and RenderSystem. */
+class PhysicsRenderBridge {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: m_staleBuffer

(cppcoreguidelines-pro-type-member-init)